src/components/media-player/media-player.component.spec.ts

```diff
@@ -0,0 +1,76 @@
+        component.displayName = '';
+        fixture.detectChanges();
+        expect(getTitle().innerText).toBeFalsy();
+    });
+
+    it('should display displayName when one is added to the markup', () => {
+        component.displayName = 'Display Name';
+        fixture.detectChanges();
+        expect(getTitle().innerText).toBeTruthy();
+    });
+
+    it('should display the filename when a file is used', (done: DoneFn) => {
+        component.displayName = '';
+        component.audioSource = 'https://uxaspects.github.io/UXAspects/assets/Ocean-Waves.mp3';
+        fixture.detectChanges();
+
+        setTimeout(() => {
+            fixture.detectChanges();
+            expect(getTitle().innerText).toBe('Ocean-Waves.mp3');
+            done();
+        }, 2000);
+    });
+
+    function getTitle(): any {
+        return nativeElement.querySelector('.player-container .audio-player .audio-file-name');
```

Comment:
This should probably return `innerText` rather than appending it to each function call.